pandas/core/generic.py

```diff
@@ -4314,10 +4314,16 @@ def get(self, key, default=None):
         2014-02-14          22.0    medium
         2014-02-15          35.0    medium
 
+        >>> df['windspeed'].get('2014-02-13')
```

Comment:
For clarity, could you assign `>>> ser = df["windspeed"]` above and then use `>>> ser.get(...`?

Reply:
I'm not sure I understand. Is that to clarify that it's a Series and not a DataFrame? Personally, I don't think thatt's unclear since it returns a scalar using a single index/key.

Reply:
Just to clarify you're calling `.get` on a Series

Reply:
Sure, might as well